packages/i18n/src/lang/fr.json

```diff
@@ -696,6 +716,10 @@
     "defaultMessage": "Trop de demandes, veuillez patienter une minute et réessayer.",
     "description": "Message displayed when number of user attempts exceeds rate limit"
   },
+  "SkGrYA": {
+    "defaultMessage": "{count, plural, =0 {0 options disponibles} =1 {1 option disponible} other {# options disponibles}}",
```

Comment:
These translations are a bit different than the English equivalents. 

1. The word _disponible_ means available where as the English equivalent word is **selected**.
2. The zero case in English uses "no" whereas French uses the number "0": should these not be the same? The easiest would be to change the English to use "0" as it wouldn't require a translation of the number.